### ISSUE TITLE
Improve question 4

### ISSUE THREAD

**Issue by bacor** [clclab/clc-labs#8]
Question 4: add instruction to adapt the function `train_on_bigrams` to do what you need for the assignment (vary learning rate and training variant and plot loss). Alternatively, add these as arguments to the function. Also: trace the loss in the train function and return it.
